_includes/about.html

```diff
@@ -26,7 +26,7 @@ <h4 class="subheading">GitHub Campus Experts Program Launch</h4>
                         </li>
                         <li>
                             <div class="timeline-image">
-                                <img class="img-circle img-responsive" src="img/badge/badge-1.png" alt="">
+                                <img class="img-responsive" src="img/badge/badge-1.png" alt="">
```

Comment:
Would it be better to remove circular plate?

Reply:
Yes, So that you could use any badge-img (not in a circular shape).